sqlglot/dialects/dialect.py

```diff
@@ -411,6 +411,9 @@ class Dialect(metaclass=_Dialect):
     ARRAY_AGG_INCLUDES_NULLS: t.Optional[bool] = True
     """Whether ArrayAgg needs to filter NULL values."""
 
+    SUPPORTS_WITHIN_GROUP_FILTER = True
+    """Whether WITHIN GROUP ( order_by ) FILTER syntax is supported for aggregate functions."""
+
```

Comment:
This is basically dead code right now, we can add it on demand later if ever needed.
```suggestion
    SUPPORTS_WITHIN_GROUP_FILTER = True
    """Whether WITHIN GROUP ( order_by ) FILTER syntax is supported for aggregate functions."""

```